lib/client/rpc/market-clients/ammMarkets.ts

```diff
@@ -78,82 +74,106 @@ export class FutarchyAmmMarketsRPCClient implements FutarchyAmmMarketsClient {
     }
   }
 
+  validateAddLiquidity(
+    ammMarket: AmmMarket,
+    quoteAmount: number,
+    maxBasAmount: number
+  ): LiquidityAddError | null {
+    const quoteAmountArg = new BN(
+      quoteAmount *
+        new BN(10).pow(new BN(ammMarket.quoteToken.decimals)).toNumber()
+    );
+    const baseAmountArg = new BN(
+      maxBasAmount *
+        new BN(10).pow(new BN(ammMarket.baseToken.decimals)).toNumber()
+    );
+    // base passed in should be ammBaseAmount honestly...
+    const baseReserve = new BN(ammMarket.baseAmount);
```

Comment:
Not sure what we're tying to do here, just wanted to double check as the values of the quoteAmount and baseAmount aren't clear what that's expected to be.